### DISCUSSION TITLE
Server OpenSubsonic API support

### DISCUSSION THREAD

**Discussion by Tolriq** [opensubsonic/open-subsonic-api#100] [General]
As discussed in an old PR, during September I'll do a PR to remove specific list of servers per API endpoints as it's no more necessary to bring attention to this project.

But in the global list of supported servers and clients, I'd like to add links to each respective servers and clients documentation page about supported endpoints if you manage one. (Would be a second link next to each client/server in https://opensubsonic.netlify.app/docs/)

So @opensubsonic/clients @opensubsonic/servers if you have such page, please list them in this post so I can directly prepare the PR with everything in it later this month.

**Comment by epoupon** [comment]
Thanks for handling this! For lms: https://github.com/epoupon/lms/blob/master/SUBSONIC.md

**Comment by paulijar** [comment]
For Nextcloud/ownCloud Music: https://github.com/owncloud/music/wiki/OpenSubsonic-API

**Comment by lachlan-00** [comment]
I'll add information on the API site here.
https://ampache.org/api/subsonic

I'm going to start updating all the extensions this week and get that written down on this URL

**Comment by gravelld** [comment]
Thanks for this @Tolriq . We've not scheduled the OpenSubsonic support at Astiga yet but will let you know when it's available.

**Comment by deluan** [comment]
Here's the current page for Navidrome: https://www.navidrome.org/docs/developers/subsonic-api

Thanks!
